app/src/test/java/com/gsandchase/mileagelogger/MileageCounterTest.java

```diff
@@ -0,0 +1,40 @@
+package com.gsandchase.mileagelogger;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class MileageCounterTest {
```

Comment:
Some example tests. Writing the tests for a class FIRST and then implementing the method ensures that you create the API that is most pleasant to use.